avionics/maintest.cpp

```diff
@@ -0,0 +1,21 @@
+//
+// Created by Quinn Ferbers on 9/29/2019.
```

Comment:
It is requested that instead of specifying your name, mark the author as UBC Rocket or UBC Rocket Avionics Team